.vscode/launch.json

```diff
@@ -6,21 +6,21 @@
     "configurations": [
         {
             "name": "Python: Current File (Integrated Terminal)",
-            "type": "python",
+            "type": "debugpy",
             "request": "launch",
             "program": "${file}",
             "console": "integratedTerminal"
         },
         {
             "name": "Python: Attach",
-            "type": "python",
+            "type": "debugpy",
             "request": "attach",
             "port": 5678,
             "host": "localhost"
         },
         {
```

Comment:
I wonder if we should leave just the Python: Django config?

Reply:
I think it is fair to remove the others and leave that one - especially since that is the only one that is mentioned in the tutorial!